mathics/core/expression.py

```diff
@@ -27,7 +27,12 @@
 )
 from mathics.core.convert.python import from_python
 from mathics.core.convert.sympy import SympyExpression, sympy_symbol_prefix
-from mathics.core.element import BaseElement, ElementsProperties, EvalMixin, ensure_context
+from mathics.core.element import (
```

Comment:
Are we having isort/black wars?   isort should be followed by black.  isort will respect black's formatting of isort'ed code. However running black and then isort, may cause isort to change the formatting when isort wants to reorder things.